### ISSUE TITLE
Some language packs missing on Play Store

### ISSUE THREAD

**Issue by rimas-kudelis** [AnySoftKeyboard/AnySoftKeyboard#4044]
### Steps to reproduce
1. Open the language pack downloads page
2. Click on Lithuanian link pointing to Play Store
3. You get a 404 page

### Expected behaviour
The language pack page should have opened

### Actual behaviour
404

**Android OS version:**
Should be irrelevant

**Device manufacturer and model:**
Samsung Galaxy S10

**List of installed add-ons (like languages, or themes):**

**Comment by SuperSentient** [comment]
This is now happening with the keyboard itself to.
Whats going on here?

**Comment by jalpar** [comment]
Is there any chance that ASK reappears on Google Play Store in the foreseeable fututre?

**Comment by billie80** [comment]
It seems that keyboard and language packs are available again.

**Comment by SuperSentient** [comment]
> It seems that keyboard and language packs are available again.

Hopefully this means that ASK will be available again on Google Play soon.

**Comment by billie80** [comment]
Well this is what I meant! Don't you find it there?

**Comment by SuperSentient** [comment]
> Well this is what I meant! Don't you find it there? https://play.google.com/store/apps/details?id=com.menny.android.anysoftkeyboard

Yep ASK is back now. :) Regardless its a good idea to have a backup keyboard just in case.
I hope ASK gets an update soon.

**Comment by rimas-kudelis** [comment]
It's back, but Google Play says the last update was in 2021.
It's difficult to understand this, considering that the repo is still relatively active.
